libraries/script-engine/src/UsersScriptingInterface.h

```diff
@@ -112,12 +119,13 @@ public slots:
 
     /**jsdoc
      * Kicks and bans a user. This removes them from the server and prevents them from returning. The ban is by user name if 
-     * available, or machine fingerprint otherwise.
+     * available and by machine fingerprint. The ban functionality can be controlled with flags.
      * <p>This function only works if you're an administrator of the domain you're in.</p>
      * @function Users.kick
      * @param {Uuid} sessionID - The session ID of the user to kick and ban.
+     * @param {number} - Preferred ban flags. <i>Bans a user by username (if available) and machine fingerprint by default.</i>
```

Comment:
instead of just having this be of type number, I think we should have JSDoc for the flags too, and then in the docs people could click the type to see the options

Reply:
Let them do the math in their heads!

I have to figure out how to get a JSDoc into the Global, I'll try and copy what's done elsewhere.